app/src/ui/changes/commit-message-avatar.tsx

```diff
@@ -121,7 +123,7 @@ export class CommitMessageAvatar extends React.Component<
 
     return (
       <Popover
-        caretPosition={PopoverCaretPosition.LeftTop}
+        caretPosition={PopoverCaretPosition.LeftBottom}
```

Comment:
I had to do this because the popup was too big and it would go beyond the boundaries of the window, and getting its content truncated.